README.md

```diff
@@ -1,181 +1,78 @@
-1. Install a language server, e.g. [pyright](CONFIG.md#pyright) via `npm i -g pyright`
-2. Install `nvim-lspconfig` via your plugin manager
-3. Add the language server setup to your init.vim. The server name must match those found in the table of contents in [CONFIG.md](CONFIG.md)
+1. Install a language server, e.g. [pyright](CONFIG.md#pyright)
+```bash
+npm i -g pyright
+```
+2. Add the language server setup to your init.vim. The server name must match those found in the table of contents in [CONFIG.md](CONFIG.md)
 ```vim
-lua << EOF
+lua << eof
 require'lspconfig'.pyright.setup{}
-EOF
+eof
 ```
-4. Open a file that is placed in a directory recognized by the server 
-(see server configuration in [CONFIG.md](CONFIG.md); e.g., for [pyright](CONFIG.md#pyright), this is 
-any directory containing ".git", "setup.py",  "setup.cfg", "pyproject.toml",
-or "requirements.txt")
-5. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling
-omnifunc completion
-
-## Built-in commands
-
-* `:LspInfo` shows the status of active and configured language servers.
-
-The following support tab-completion for all arguments:
-
-* `:LspStart <config_name>` Start the requested server name. Will only succesfully start if the command detects a root directory matching the current config. Pass `autostart = false` to your `.setup{}` call for a language server if you would like to launch clients solely with this command. Defaults to all servers matching current buffer filetype.
-* `:LspStop <client_id>` Defaults to stopping all buffer clients. 
-* `:LspRestart <client_id>` Defaults to restarting all buffer clients.
-
-## Usage
-
-**All provided examples are in Lua,** see `:help :lua-heredoc` to use Lua from your init.vim,
-or the quickstart above for an example of a lua heredoc.
-
-Each config provides a `setup()` function to initialize the server with reasonable default
-initialization options and settings, as well as some server-specific commands. This is 
-invoked in the following form, where `<server>` corresponds to the language server name
-in [CONFIG.md](CONFIG.md).
-
-`setup()` takes optional arguments <config>, each of which overrides the respective 
-part of the default configuration. The allowed arguments are detailed [below](#setup-function).
-```lua
-require'lspconfig'.<server>.setup{<config>}
+3. Create a project, this project must contain a file matching the root directory trigger. See [Automatically launching language servers](#Automatically-launching-language-servers) for additional info.
+```bash
+mkdir test_python_project
+cd test_python_project
+git init
+touch main.py
 ```
+4. Launch neovim, the language server will now be attached and providing diagnostics (see `:LspInfo`)
+`nvim main.py`
+5. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling omnifunc completion
 
-### Example: using the defaults
+## Automatically launching language servers
```

Comment:
I don't think this heading is the best choice here. It implies that you need to do something to automatically launch a language server. Maybe change it to ``The project root and the `root_dir` pattern``?

Reply:
My issue with this is it's not attention grabbing. People might breeze past a discussion on "project roots" thinking it's some advanced feature but this is 99% of the issues with language servers